ctdcal/convert.py

```diff
@@ -130,14 +107,11 @@ def convertFromSBEReader(sbeReader, debug=False):
     raw_df.index.name = 'index'
     raw_df = raw_df.apply(pd.to_numeric, errors="ignore")
 
-    #debugPrint("Raw Data Types:", raw_df.dtypes)
-    #debugPrint("Raw Data:", raw_df.head)
-
     # Retrieve Config data
     rawConfig = sbeReader.parsed_config()
 
     # The meta data field needs to be processed seperately and then joined with the converted_df
-    debugPrint("Building meta data dataframe... ", end='')
+    print("Building meta data dataframe... ")
```

Comment:
Note that the "debugPrint" was printing to stderr, I would highly recommend spending some time setting up the logging system. See https://docs.python.org/3/library/logging.html

Here is an example of someone at CCHDO who did some significant customization https://github.com/cchdo/libcchdo/blob/master/libcchdo/log.py

If you set things up right, the logging messages will tell you where in the library things are being emitted and that can be super useful.

Reply:
Yes, great point. I have implementing logging as one of the next high priority projects after the code pruning is completed.